tests/monitoring/monitoring.go

```diff
@@ -640,8 +645,70 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 		})
 	})
+
+	Context("Warning alerts", func() {
+		alerts := []string{
+			"KubeVirtComponentExceedsRequestedCPU",
+			"KubeVirtComponentExceedsRequestedMemory",
+		}
+
+		BeforeEach(func() {
+			virtClient, err = kubecli.GetKubevirtClient()
+			util.PanicOnError(err)
+
+			scales = make(map[string]*autoscalingv1.Scale, 1)
```

Comment:
I must admit that the pattern of allocating the map of `*autoscalingv1.Scale` outside of `backupScale` and `revertScale` is weird, why can't you pass list of names of what you want to backup/revert, and let these helper functions to manage the `scales` map inside?

Reply:
moved the scaling logic to a new file, and added a new method `setScales` that takes care of that logic